src/Data/Decimal/Extras.js

```diff
@@ -0,0 +1,11 @@
+// module Data.BigInt.Extras
+
+var Decimal = require("decimal.js");
+
+exports.truncated = function(x) {
+  return x.truncated();
+}
+
+exports.isInteger = function(x) {
+  return x.isInteger();
+}
```

Comment:
I would like for these modules to be in a PreciseDateTime internal module. This isn't a public API we are supporting.

Reply:
Should we just make a PR to the decimal library to support this?

Reply:
I think that should be the goal. This at least let us get this far without forking and dealing with that overhead, but maybe now is the right time.

Reply:
PR up here: https://github.com/Ambrosia/purescript-decimal/pull/1

In the meantime, should I switch to my fork of `purescript-decimal`, or move this to an internal module?